docroot/sites/housing.uiowa.edu/modules/housing_core/templates/field--paragraph--field-housing-floor-plan-pano--housing-floor-plan.html.twig

```diff
@@ -39,7 +39,8 @@
 #}
 
 {% for item in items %}
-  <div class="embed-responsive embed-responsive-16by9">
+  {{ attach_library('uids_base/media') }}
+  <div class="media--16-9">
```

Comment:
```suggestion
  <div class="media media--16-9">
```
Seems to resolve https://github.com/uiowa/uiowa/pull/6120#issuecomment-1447197349